db/migrate/20200108180542_create_sample_types.rb

```diff
@@ -27,6 +27,7 @@ def populate_sample_types!
       'CNS Infections': [
         { name: 'Brain' },
         { name: 'Cerebrospinal Fluid', human_only: true },
+        { name: 'Central Nervous System' },
```

Comment:
I opportunistically fixed this following discussion with Katrina . It will work as long as this PR is deployed along with the previous 😸